uberfire-workbench/uberfire-workbench-client/src/main/java/org/uberfire/client/mvp/ActivityBeansCache.java

```diff
@@ -119,6 +122,61 @@ private void put(final SyncBeanDef<Activity> activityBean,
                            activityBean);
     }
 
+    void registerGwtEditorProvider() {
+        nativeRegisterGwtEditorProvider();
+    }
+
+    private native void nativeRegisterGwtEditorProvider() /*-{
```

Comment:
Do you mind moving the native code to a separate class? Having it here deceases the readability of the class